### ISSUE TITLE
Upgrading the manifest

### ISSUE THREAD

**Issue by omid-verneek** [argoproj-labs/argocd-vault-plugin#267]
Installing the plugin is very challenging and hard to follow. Currently, the manifest folder seems to be not in a good place and out of date. It would be great to take three quick actions to improve the operation experince:

1. Upgrading the version of the plugin in the manifest (or changing the manifest to use the latest release). The manifest version currently is: `1.1.1`
2. Adding a value file to add the secrets needed to install the plugin. Also, the manifests don't include the following which I found from this blog: https://itnext.io/argocd-secret-management-with-argocd-vault-plugin-539f104aff05
```yaml
 envFrom:
    - secretRef:
        name: argocd-vault-plugin-credentials
``` 
3. Creating more docs around how to use the manifests to install the plugin.

**Comment by jkayani** [comment]
@omid-verneek You're right, those manifests haven't been touched in a while - sorry about that. We'll try to get that cleaned up soon, but feel free to beat us to it and open a PR with some fixes.

**Comment by werne2j** [comment]
Thanks @omid-verneek for looking for ways to improve the documentation! The official documentation on installing the plugin is https://argocd-vault-plugin.readthedocs.io/en/stable/installation. The manifest folder is a convenience at definitely out of date.

Feel free to contribute documentation that you think would help make the installation a simpler process and @jkayani will take a look into it as well.

**Comment by jkayani** [comment]
Updated manifests here: https://github.com/argoproj-labs/argocd-vault-plugin/tree/main/manifests and docs here: https://argocd-vault-plugin.readthedocs.io/en/latest/installation/. Hopefully it's more clear.
